### PR TITLE
Moves shuttle related logs into a new log file

### PR DESCRIPTION
**Pull request by 81Denton**
This PR moves shuttle related logs into a new log file (shuttle.log). It also adds logging to shuttle consoles and aux bases/beacons. Shuttle gibbing is logged in the attack log.

## Changelog
:cl: Denton
admin: Moved shuttle related logs into a new log file called shuttle.log. 
admin: Added logging for shuttle consoles, aux bases/beacons and shuttle gibbing (last one is in the attack log).
/:cl:

### CODE REVIEW

**Comment by optimumtact** [comment]
make a new file, game log is already too busy

**Comment by 81Denton** [comment]
I'll look into it

**Comment by ShizCalev** [comment]
move all shuttle related stuff to that file too if you wouldn't mind (shuttle manipulator, emergency console, ect)

**Comment by 81Denton** [comment]
I was thinking of a shuttle log too, let me look at some recent game logs and see if I find anything else that would fit

**Comment by 81Denton** [comment]
@ShizCalev Shuttle manipulator currently uses the admin log since it's admin only. I think it would make more sense to leave it that way.

**Comment by ShizCalev** [comment]
I'd say make a copy there too, but I guess it's fine in just badmin log

**Comment by 81Denton** [comment]
Alright, I moved all shuttle related logging I could find (minus admin actions) to a new shuttle.log file

**Review by ShizCalev — DISMISSED** [review]

File: code/__DEFINES/logging.dm [hunk 1]
```diff
@@ -44,7 +45,7 @@
 #define INDIVIDUAL_COMMS_LOG		(LOG_PDA | LOG_CHAT | LOG_COMMENT | LOG_TELECOMMS)
 #define INDIVIDUAL_OOC_LOG			(LOG_OOC | LOG_ADMIN)
 #define INDIVIDUAL_OWNERSHIP_LOG	(LOG_OWNERSHIP)
-#define INDIVIDUAL_SHOW_ALL_LOG		(LOG_ATTACK | LOG_SAY | LOG_WHISPER | LOG_EMOTE | LOG_DSAY | LOG_PDA | LOG_CHAT | LOG_COMMENT | LOG_TELECOMMS | LOG_OOC | LOG_ADMIN | LOG_OWNERSHIP | LOG_GAME)
+#define INDIVIDUAL_SHOW_ALL_LOG		(LOG_ATTACK | LOG_SAY | LOG_WHISPER | LOG_EMOTE | LOG_DSAY | LOG_PDA | LOG_CHAT | LOG_COMMENT | LOG_TELECOMMS | LOG_OOC | LOG_ADMIN | LOG_OWNERSHIP | LOG_GAME | LOG_SHUTTLE)
```
Comment (ShizCalev) [inline code/__DEFINES/logging.dm]: actually, would you mind adding the rest of the missing ones? ie log mecha, virus, ect

Comment (81Denton) [reply code/__DEFINES/logging.dm]: ok, got them

File: code/modules/shuttle/on_move.dm [hunk 2]
```diff
@@ -34,6 +34,7 @@ All ShuttleMove procs go here
 				M.visible_message("<span class='warning'>[shuttle] slams into [M]!</span>")
 				SSblackbox.record_feedback("tally", "shuttle_gib", 1, M.type)
 				M.gib()
+				log_attack("[key_name(M)] was shuttle gibbed by [shuttle].")
```
Comment (Cyberboss) [inline code/modules/shuttle/on_move.dm]: `gib` deletes the mob, move this up

**Review by Cyberboss — CHANGES_REQUESTED** [review]

File: code/modules/shuttle/on_move.dm (hunk 2 above)
Comment (81Denton) [reply code/modules/shuttle/on_move.dm]: fixed it

**Review by Cyberboss — APPROVED** [review]

**Pull request merged**